regression-test/data/nereids_function_p0/array/test_array_map.csv

```diff
@@ -0,0 +1,11 @@
+// this is data case file for array<map<largeInt:string,datetime:decimal<76,56>, char:double, ipv4:ipv6>> type, contains normal, corner, exceptional data cases
```

Comment:
what about array<json> case

Reply:
we do not support create data type array nested json